Buildings/Experimental/OpenBuildingControl/ASHRAE/G36/AtomicSequences/EconModulationMultiZone.mo

```diff
@@ -0,0 +1,194 @@
+  CDL.Interfaces.RealInput uOutDamPosMin(min=0, max=1, unit="1")
+    "Minimum economizer damper position limit as returned by the EconDamperPositionLimitsMultiZone sequence"
+    annotation (Placement(transformation(extent={{-160,-120},{-120,-80}}),
+        iconTransformation(extent={{-120,-30},{-100,-10}})));    //fixme: add quantity for ALL damper positions?
+  CDL.Interfaces.RealInput uOutDamPosMax(min=0, max=1, unit="1")
+    "Maximum economizer damper position limit as returned by the EconEnableDisableMultiZone sequence. 
+    If the economizer is disabled, this value equals uOutDamPosMin"
+    annotation (Placement(transformation(extent={{-160,-90},{-120,-50}}),
+        iconTransformation(extent={{-120,0},{-100,20}})));
+  CDL.Interfaces.RealInput uRetDamPosMin(min=0, max=1, unit="1")
+    "Minimum return air damper position limit as returned by the EconEnableDisableMultiZone sequence"
+    annotation (Placement(transformation(extent={{-160,40},{-120,80}}),
+        iconTransformation(extent={{-120,-100},{-100,-80}})));
+  CDL.Interfaces.RealInput uRetDamPosMax(min=0, max=1, unit="1")
+    "Maximum return air damper position limit as returned by the EconEnableDisableMultiZone sequence."
+    annotation (Placement(transformation(extent={{-160,80},{-120,120}}),
+        iconTransformation(extent={{-120,-70},{-100,-50}})));
+
+  CDL.Interfaces.RealOutput yOutDamPos(min=0, max=1, unit="1") "Economizer damper position"
+    annotation (Placement(transformation(extent={{120,-30},{140,-10}}),
+        iconTransformation(extent={{100,-30},{120,-10}})));
+  CDL.Interfaces.RealOutput yRetDamPos(min=0, max=1, unit="1") "Return air damper position"
+    annotation (Placement(transformation(extent={{120,10},{140,30}}),
+        iconTransformation(extent={{100,10},{120,30}})));
+
+  CDL.Continuous.LimPID damPosController(
+    Td=0.1,
+    Nd=1,
+    controllerType=Buildings.Experimental.OpenBuildingControl.CDL.Types.SimpleController.PI,
+    Ti=300,
+    yMax=yConSigMax,
+    yMin=yConSigMin,
+    k=controllerGain)
+    "Contoller that outputs a signal based on the error between the measured SAT and SAT cooling setpoint"
+    annotation (Placement(transformation(extent={{-80,0},{-60,20}})));
+  CDL.Continuous.Line outDamPos(limitBelow=true, limitAbove=true)
+    "Damper position is linearly proportional to the control signal between signal limits"
+    annotation (Placement(transformation(extent={{60,-40},{80,-20}})));
+  CDL.Continuous.Line retDamPos(limitBelow=true, limitAbove=true)
+    "Damper position is linearly proportional to the control signal between signal limits"
+    annotation (Placement(transformation(extent={{60,60},{80,80}})));
+
+protected
+  CDL.Continuous.Constant outDamMinLimSig(final k=damPosController.yMin)
+    "Minimal control loop signal for the outdoor air damper."
+    annotation (Placement(transformation(extent={{-20,-20},{0,0}})));
+  CDL.Continuous.Constant outDamMaxLimSig(k=outDamConSigMax)
```

Comment:
The one above is the controller min signal, which is final. The one below is usually set to 0.5, but that value is not strictly fixed. I could imagine that some economizers could operate better if, for example, we allocate larger fraction of the control signal to outdoor air damper, compared to the return air damper. This is if ```fixed``` should precede a parameter which will never require changes.